File: dayOne/levelOne/python/palindromic_array/palindromic_array.py
def check_if_array_is_a_palindrome(numbers):

    begining = 0;
    end = len(numbers) - 1;

    while begining < end:

        if numbers[begining] != numbers[end]: 
            return False;

    begining+=1;
    end-=1;


    return True;

File: dayOne/levelOne/java/palindromicArray/palindromicArray.java
public class palindromicArray {

    public static boolean checkIfArrayIsAPalindrome(int[] numbers) {

        int begining = 0;
        int end = numbers.length - 1;

        while (begining < end) {

            if (numbers[begining] != numbers[end]) {
                return false;
            }

            begining++;
            end--;
        }

        return true;
    }

}
